test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -671,4 +673,17 @@ public void canTrackBellydancerPickpocket() {
     parseCombatData("request/test_fight_bellydancing_pickpocket_3.html");
     assertEquals(3, Preferences.getInteger("_bellydancerPickpockets"));
   }
+
+  @ParameterizedTest
+  @ValueSource(ints = {7, 12})
+  public void canTrackDesignerSweatpants(int expectedSweat) {
+    var cleanups =
+        new Cleanups(
+            equip(EquipmentManager.PANTS, "designer sweatpants"), setProperty("sweat", 10));
+
+    try (cleanups) {
+      parseCombatData("request/test_fight_designer_sweatpants_" + expectedSweat + ".html");
```

Comment:
These files are now named based on a base sweat of 10. You'd be better here to make a CsvSource of the HTML name (which is renamed to be like "test_fight_gain_x_sweat" etc) and the expectedSweat

Reply:
Implemented